misc/python/materialize/mzcompose/services.py

```diff
@@ -541,7 +541,6 @@ def __init__(
             healthcheck = {
                 # init_success is a file created by the Cockroach container entrypoint
                 "test": "[ -f init_success ] && curl --fail 'http://localhost:8080/health?ready=1'",
-                "timeout": "5s",
```

Comment:
I understand that the default timeout will now apply instead? What value is the default?

Reply:
The default is 30 seconds, that is , docker will wait 30 seconds for the curl above to return something.

Reply:
Thanks! 👍